src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -1892,6 +1908,29 @@ public bool IsWindowRotating()
             return ret;
         }
 
+        /// <summary>
+        /// Set the window use partial update or not.
+        /// </summary>
+        /// <param name="enabled">If window enable partial update or disable.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetPartialUpdateEnabled(bool enabled)
```

Comment:
You already defined a property for this : `bool PartialUpdate`
Is there any reason to define it as methods too? It seems redundant.

Reply:
Hi. There are several reason about this API.

1. Most of Windows API already implements similar patterns
2. I just want to 'hide' the Interop API using for `Property` level.
3. For product release sight, `hidden`-to-`public` is more easier than `internal`-to-`public` .

Reply:
I agree with 2&3.

For 1, it seems not common to have both public property and methods for the same functionality.
And even if it is true, I think they have at least same API names
```C#
bool PartiaUpdateEnabled;
void SetPartiaUpdateEnabled(bool);
bool IsPartiaUpdateEnabled();
```
Or
```C#
bool PartiaUpdate;
void SetPartiaUpdate(bool);
bool IsPartiaUpdate();
```

Reply:
Let me add one more thing.
For property name, it is better to have a prefix `Is` for a suffix `Enabled` as a set.
`bool IsPartialUpdateEnabled`
For example,
https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.visualelement.isenabled?view=net-maui-7.0
And NUI has simalr property name patterns.

Reply:
Append keyword 'Enabled' at Property feel bad to me. I make API name as `SetPartialUpdate`.

Reply:
I understand your feelings to a specific name for a property.  :smile:
NUI already have properties named `IsXXXEnabled` and also MAUI has some. You can give more chance to `Enabled` suffix.

Reply:
Anyway, It is not a big deal but It's still hard to find a case having both public property and method in NUI for me.
Even if there is a case I believe it was not intentional.
I found a few cases but their methods are `internal` normally. In that case, it seems that they don't have a plan to be a `public`. (And I think they need to be a private)

Do you really need both `property` and `method`?

Reply:
If you don't want a property, you can remove property and provide only methods anyway.